### PR TITLE
Uploaded Assignment 9

### PR DESCRIPTION
**Pull request by Prashant698**

### CODE REVIEW

**Review by Pawan1005 — APPROVED** [review]

File: prashant/HTML/Assinment9/styles.css [hunk 1]
```diff
@@ -0,0 +1,336 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+.row {
+    /* border: 1px solid greenyellow; */
+}
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assinment9/styles.css:8]: remove .row class if not needed

File: prashant/HTML/Assinment9/styles.css [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+.center {
+    display: flex;
+    align-items: center;
+    /* justify-content: center; */
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assinment9/styles.css:12]: remove comment

File: prashant/HTML/Assinment9/styles.css [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    margin-top: 10px;
+}
+.row::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assinment9/styles.css:20]: remove extra line

File: prashant/HTML/Assinment9/styles.css [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+[class*="col-"] {
+    float: left;
+    padding: 1px;
+}
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assinment9/styles.css:25]: remove extra line

File: prashant/HTML/Assinment9/styles.css [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+}
+.col-11 {
+    width: 91.66%;
+}
+.col-12 {
+    width: 100%;
+}
+/* .center {
+    align-self: center;
+} */
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assinment9/styles.css:64]: remove comment

**Review by sourabh-kanwade — APPROVED** [review]

**Pull request merged**